bindings/ocaml/EverCrypt.ml

```diff
@@ -111,12 +111,14 @@ end
 
 module Chacha20_Poly1305 : Chacha20_Poly1305 =
   Make_Chacha20_Poly1305 (struct
+    let reqs = []
```

Comment:
I would add a comment here saying that run-time checks are already performed by the evercrypt API and that as such the OCaml bindings layer does not need to check for anything